topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -58,19 +58,19 @@ This tutorial assumes you are comfortable getting data into Galaxy, running jobs
 
 # VGP assembly workflow structure
 
-The VGP assembly pipeline has a modular organization, consisting in five main subworkflows (fig. 1), each one integrated by a series of data manipulation steps. Firstly, it allows the evaluation of intermediate steps, which facilitates the modification of parameters if necessary, without the need to start from the initial stage. Secondly, it allows to adapt the workflow to the available data (e.g. not to include the module corresponding to bionano when the data corresponding to this technology is not available).
+The VGP assembly pipeline has a modular organization, consisting of five main subworkflows (fig. 1), each one integrated by a series of data manipulation steps. This modular nature allows for the quality control assessment of intermediate steps, so that one can modify parameters (if necessary) without having to start the assembly from scratch. Additionally, this allows one to adapt the pipeline to the avilable data (*e.g.*, if Bionano data is not available, then it is not necessary to run the Bionano scaffolding workflow). 
```

Comment:
```suggestion
The VGP assembly pipeline has a modular organization, consisting of five main sub-workflows (fig. 1), each one integrated by a series of data manipulation steps. This modular nature allows for the quality control assessment of intermediate steps, so that one can modify parameters (if necessary) without having to start the assembly from scratch. Additionally, this allows one to adapt the pipeline to the available data.
```

Reply:
The last sentence was removed because it was a bit redundant.